watsononlinestore/watson_online_store.py

```diff
@@ -86,29 +89,32 @@ def __init__(self, bot_id, slack_client,
 
         # IBM Watson Conversation
         self.conversation_client = conversation_client
-        self.discovery_client = discovery_client
         self.workspace_id = self.setup_conversation_workspace(
             conversation_client, os.environ)
 
         # IBM Cloudant noSQL database
         self.cloudant_online_store = cloudant_online_store
 
-        # IBM Discovery Service
+        # IBM Watson Discovery Service
+        self.discovery_client = discovery_client
         self.discovery_data_source = os.environ.get(
-            'DISCOVERY_DATA_SOURCE')
-        self.discovery_environment_id = os.environ.get(
-            'DISCOVERY_ENVIRONMENT_ID')
-        self.discovery_collection_id = os.environ.get(
-            'DISCOVERY_COLLECTION_ID')
+            'DISCOVERY_DATA_SOURCE', DiSCOVERY_IBM_STORE)
```

Comment:
actually, type is in the constant definition. I will change that, and all of the references...